tests/core/evm/vm/test_message.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import evm.vm.message as m
+
+message_object = m.Message(
```

Comment:
Typically it's best to avoid globals in test modules (an really in general).  would suggest the following change.

Instead of a global object, define a local "factory" function that you can use to easily generate message objects.  The following is probably a good starting point.

```python
def _create_message(gas=1, gas_price=1, to=b"...", sender=b"...", data=b"", code=b"", origin=b"..."):
    return Message(gas=gas, gas_price=gas_price, ...)
```

For `to`, `sender`, and `origin`, I would recommend using a non-zero, non-empty address such as `\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff`.  This is clearly a test address, but also a valid address.

Next, at the top of this module, you need to add `from __future__ import unicode_literals`.  This forces python2 to behave the same as python3 in terms of the string types it infers.